src/java.base/linux/classes/jdk/internal/platform/CgroupSubsystemFactory.java

```diff
@@ -149,6 +149,10 @@ public static Optional<CgroupTypeResult> determineType(String mountInfo,
             case MEMORY_CTRL:   infos.put(MEMORY_CTRL, info); break;
             case BLKIO_CTRL:    infos.put(BLKIO_CTRL, info); break;
             case PIDS_CTRL:     infos.put(PIDS_CTRL, info); break;
+            default:
+                // There are some controllers (such as freezer) that Java doesn't
+                // care about. Just ignore them. These are not considered in the
+                // anyCgroupsV1Controller/anyCgroupsV1Controller checks.
```

Comment:
It's not clear why this `default` is necessary. Could we just add the comment like so:

```
// Intentional fall-through. There are controllers (such as freezer) that
// Java doesn't care about. Just ignore them. Only listed controllers are
// considered in the anyCgroupsV1Controller/anyCgroupsV2Controller checks.
```

Reply:
My bad. How about `Intentional incomplete switch. There are ...`? Anyway, why is the empty `default` case needed other than for the comment?

Reply:
To be, the `default:` switch is a clear indication that "everything else comes here". So you won't be confused whether the comment is related to the last line just above the comment.

Reply:
It confused me, fwiw. Anyway up to you. It's not super important.

Reply:
works for me. +1. Note the typo `anyCgroupsV1Controller/anyCgroupsV2Controller` not **V1** twice.

Reply:
Oops, I'll fixed that. Thanks!